plugins/go.mod

```diff
@@ -0,0 +1,33 @@
+module github.com/apiclarity/apiclarity/plugins
+
+go 1.17
+
+require (
+	github.com/apiclarity/apiclarity/plugins/api v0.0.0-20211209155154-948acf2ec933
```

Comment:
It is a bit tricky to use replace here, since you count on this that you will have the same folder structure in the Dockerfile build. In tyk, we have to rely on their folder structure since they run the build from a script, and tyk is using the plugin go.mod
This is why I dont use replace

Reply:
We can do utils/common package

Reply:
> It is a bit tricky to use replace here, since you count on this that you will have the same folder structure in the Dockerfile build

What is the problem to create the same structure in the docker build? we do it in all other docker files I think (also in the backend).
Not using replace here will make build issues when you are changing the plugins api swagger